managed/services/management/node.go

```diff
@@ -121,9 +131,16 @@ func (s *NodeService) Register(_ context.Context, req *managementpb.RegisterNode
 			CreateNodeExporter(tx.Querier, pmmAgent.AgentID, nil, isPushMode(req.MetricsMode), req.DisableCollectors,
 				pointer.ToStringOrNil(req.AgentPassword), "")
 		return err
-	}); e != nil {
+	})
+	if e != nil {
 		return nil, e
 	}
 
+	apiKeyName := fmt.Sprintf("pmm-agent-%s-%d", req.NodeName, rand.Int63())
```

Comment:
Can you please confirm we initialize `rand` with `seed` or in other ways so we actually get random integers when `managed` runs?

Reply:
done init rand seed in main